src/views/bridge-confirmation/bridge-confirmation.view.tsx

```diff
@@ -75,6 +75,7 @@ const BridgeConfirmation: FC = () => {
   const [estimatedFee, setEstimatedFee] = useState<AsyncTask<BigNumber, string>>({
     status: "pending",
   });
+  const feePollingInterval = useRef<NodeJS.Timer>();
```

Comment:
We shouldn't make any reference to `NodeJS` types in the browser. This should be `useRef<number>`, and then later you should use `window.setInterval` instead of just `setInterval`.

Reply:
Good catch! I didn't know that typing this as `NodeJS.Timer` could imply runtime errors since it's a number in the browser. Fixed.